docs/rules/object-shorthand.md

```diff
@@ -83,12 +83,22 @@ var foo = {
 ## Options
 
 The rule takes an option which specifies when it should be applied. It can be set to
-"always", "properties", "methods", or "never". The default is "always".
+"always", "properties", "methods", "never", or "avoid-quotes". The default is "always".
```

Comment:
After looking at this, I don't think this is the right approach. None of the other options have to do with quotes.